applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -191,7 +191,7 @@ Vector& ElasticIsotropic3D::CalculateValue(
         r_flags.Set( ConstitutiveLaw::COMPUTE_STRESS, true );
 
         // We compute the stress
-        this->CalculateMaterialResponseCauchy(rParameterValues);
+        ElasticIsotropic3D::CalculateMaterialResponseCauchy(rParameterValues);
```

Comment:
wont this prevent deriving from this class and overridding `CalculateMaterialResponseCauchy`?

Reply:
This means that the derived classes should implement the method too

Reply:
(But I added the calculate basically because of the finite strain laws, which are right now just 3D)